src/apps/config/support/snabb-softwire-v1.lua

```diff
@@ -12,6 +13,51 @@ local ctable = require('lib.ctable')
 local cltable = require('lib.cltable')
 local path_mod = require('lib.yang.path')
 local generic = require('apps.config.support').generic_schema_config_support
+local binding_table = require("apps.lwaftr.binding_table")
+
+
+local binding_table_instance
+local function get_binding_table(conf)
```

Comment:
For the above reason I think we can and probably should do everything on a configuration instance.